### ISSUE TITLE
Brush center for snap to grid

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#723]
We should have an option to specify what is the center of the brush so we can align it with the grid corner.

**Comment by huwpascoe** [comment]
Agreed. I wanna quickly sketch out tiles but the current grid offset is impossible to work with!

**Comment by mljli** [comment]
@dacap  Are you still working on this? I wonder why this isn't a P0 task. It really confused me when I used the snap to grid tool for the first time. And I guess it also confuses other people and causes inconveniences. I'm new to aseprite but if it's not hard, I'll look at the code and see if I can help with it.

**Comment by Gasparoken** [comment]
Internal use: reported https://igarastudio.zendesk.com/agent/tickets/4563

**Comment by hsandt** [comment]
I just tried to enable Snap to Grid with a Grid of 2x2 to test, and when using a brush of same size (2px square) and clicking near the bottom-right of a grid cell, but inside that grid cell, said grid cell is filled, not the grid cell in the bottom-right. So isn't that fixed? Or are we talking about the brush *preview* which doesn't match where we actually draw, and yes that would be a nice feature too, but it seems the described issue is something else?

**Comment by ocio28** [comment]
still no solution?
